SYCL/Basic/context-with-multiple-devices.cpp

```diff
@@ -20,11 +20,6 @@ int main() {
   auto DeviceList =
       sycl::device::get_devices(sycl::info::device_type::accelerator);
 
-  // remove host device from the list
```

Comment:
What a weird test: ask for accelerator devices and then remove `host` from them. I wonder why did the test expect `host` to even be in that list?

Reply:
That's a very good question. It may have been restricted later and whoever did that just forgot to remove the redundant filtering. Or maybe it was a copy-paste mistake.